### DISCUSSION TITLE
Most used addons according to ~100 feedback messages

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#3635] [Discussion]
Data is since Aug 26 (v1.19.1) until Oct 10 (v1.20.2)

![](https://cdn.discordapp.com/attachments/755073703393624177/896794912887038022/enabled_addons_with_tags_4.png)

**Comment by Cool-Scratcher** [comment]
These are some interesting statistics. Semicolon is the least enabled addon? Editor stage left is the least enabled *featured* addon? Hide flyout is the least enabled *recommended* addon?? And resizable comment input is the most *disabled* enabled by default addon????

**Reply by WorldLanguages** [reply]
Yeah.
However I wouldn't trust these too much. It's hard to believe that `user-id` is used by 45% of users. It's practically useless. People that send feedback apparently are more likely to enable the newest addons.

**Reply by GrahamSH-LLK** [reply]
Nobody is surprised about semicolon

**Comment by Dhuls-Scratch** [comment]
Ago?
It might be more accurate if we collected aggregated data about which addons were used, although i'm not sure if many people would like that.

**Reply by WorldLanguages** [reply]
Ago → Aug (Spanish :P)
We probably don't want telemetry.
